honeybee_vtk/model.py

```diff
@@ -153,6 +155,15 @@ def cameras(self):
         """List of Camera objects attached to this Model object."""
         return self._cameras
 
+    def get_modeldataset_from_string(self, text: str) -> ModelDataSet:
+        if text not in AcceptedValues.names.value:
```

Comment:
You just need a list or if you want to have a quicker check then create a `set` of values. An enumeration is an overkill.

Reply:
I thought I will add other items here in future such as acceptable delimiters. I turned this into a tuple for now to comply with the comment

Reply:
Sounds good.